RevenueCatUI/CustomerCenter/Abstractions/CustomerCenterPurchasesType.swift

```diff
@@ -23,7 +23,9 @@ protocol CustomerCenterPurchasesType: Sendable {
     var isSandbox: Bool { get }
 
     @Sendable
-    func customerInfo() async throws -> CustomerInfo
+    func customerInfo(
+        fetchPolicy: CacheFetchPolicy
```

Comment:
I wanted to make this default to `.default`, but Swift doesn't allow default values in protocols 😅